roles/jenkins_master/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - role: certificate_authority
+  - role: repositories
+  # TODO: consider adding jenkins_common role as a dependency
```

Comment:
Opening an issue is my thought as well, I just wanted to be sure that we actually need this.